src/components/mobile-menu-container/components/mobile-user-disclosure/mobile-user-disclosure.module.css

```diff
@@ -0,0 +1,99 @@
+.root {
+	/* composition */
+	composes: hds-surface-base from global;
+
+	/* properties */
+	background-color: var(--token-color-surface-faint);
+	border-radius: 6px;
+	margin-bottom: 24px;
+}
+
+.activator {
+	/* composition */
+	composes: g-focus-ring-from-box-shadow from global;
+
+	/* properties */
+	align-items: center;
+	background-color: transparent;
+	border-radius: 6px;
+	border: none;
+	display: flex;
+	gap: 8px;
+	justify-content: space-between;
+	padding: 16px;
+	outline: transparent;
+
+	/* aka disclosure is open */
+	&[aria-expanded='true'] {
```

Comment:
💭 It would be nice to add a transition animation to the content opening also. Right now it feels pretty jumpy. That's definitely a polish item though.

Reply:
Good call! We need to do this for `AccordionDisclosure` as well, so I've added a ticket for us to handle this for (hopefully) all `Disclosure` components: https://app.asana.com/0/1202097197789424/1202689706806169/f